tomviz/PipelineExecutor.cxx

```diff
@@ -451,7 +450,7 @@ void DockerPipelineExecutor::cancel(std::function<void()> canceled)
           });
 }
 
-bool DockerPipelineExecutor::cancel(Operator* op)
+bool DockerPipelineExecutor::cancel(Operator*)
```

Comment:
I would prefer we didn't touch this, as it will cause conflicts with the existing docker PR.